Source/WebCore/accessibility/AXTextMarker.cpp

```diff
@@ -547,7 +547,9 @@ int AXTextMarker::lineIndex() const
         // Start from a line end, so that subsequent calls to nextLineEnd() yield a new line.
         // Otherwise if we started from the middle of a line, we would count the the first line twice.
         auto nextLineEndMarker = currentMarker.nextLineEnd();
-        TEXT_MARKER_ASSERT_DOBULE(nextLineEndMarker.lineID() == currentMarker.lineID(), nextLineEndMarker, currentMarker);
+        TEXT_MARKER_ASSERT_DOUBLE(nextLineEndMarker.lineID() == currentMarker.lineID(), "lineIndex", nextLineEndMarker, currentMarker);
+        if (nextLineEndMarker.lineID() != currentMarker.lineID())
+            return -1;
```

Comment:
If this condition fails, the worst that would happen is that the computed `index` is off-by-one, right? I feel like that's better than returning `-1` 🤔